README.md

```diff
@@ -53,7 +53,7 @@ See [Project Logistics](projects/logistics.md) for important dates and deliverab
 * [Datalogue](projects/datalogue.md)
 * [Screencast Platform](projects/screencast-platform.md)
 * [Automated News Platform](projects/automated-news-platform.md) - **Team:** [Bill Marino](people/bill-marino.md)
-* [Sleep Regulation App](projects/sleep-regulator.md)
+* [Sleep Regulation App](projects/sleep-regulator.md) - **Team:** [Claire Opila] (people/claire-opila.md), [Francesco Perera] (people/francesco-perera.md), [Praveen Gupta](people/praveen-gupta.md)
```

Comment:
There're those extra spaces after the names of Claire and Francesco, ... the links seem to work, but can we clean them up please?